digitalocean/terraform/backend.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  backend "s3" {
```

Comment:
The examples should also easily be runnable by external customers. Therefor remove the S3 backend here. We should only use the default file backend for the state.